packer-rhel8-ami.json

```diff
@@ -0,0 +1,68 @@
+{
+  "builders": [{
+    "type": "amazon-ebs",
+    "access_key": "{{user `aws_access_key`}}",
+    "secret_key": "{{user `aws_secret_key`}}",
+    "region": "{{user `region`}}",
+    "encrypt_boot": false,
+    "ena_support": true,
+    "source_ami_filter": {
+      "filters": {
+        "virtualization-type": "hvm",
+        "name": "RHEL-8.4.0_HVM-*x86_64*",
```

Comment:
Don't hate me, but it looks like RHEL 8.5 GA came out in November (after Dan did this work). What do you think about trying it out?

Reply:
Sure, why not.